README.md

```diff
@@ -1,24 +1,11 @@
 # hyper-content-for
 
-Allows you to store HTML for use (and re-use) later.
```

Comment:
Considering ES6 will bring “yield” to the table, albeit in a slightly different fassion, it might be familiar, and it might be confusing. :stuck_out_tongue_closed_eyes: